src/fu-util.c

```diff
@@ -1325,15 +1325,11 @@ fu_util_get_device_or_prompt(FuUtilPrivate *priv, gchar **values, GError **error
 
 	/* get device to use */
 	if (g_strv_length(values) >= 1) {
-		g_autoptr(GError) error_local = NULL;
 		if (g_strv_length(values) > 1) {
 			for (guint i = 1; i < g_strv_length(values); i++)
 				g_debug("Ignoring extra input %s", values[i]);
 		}
-		dev = fu_util_get_device_by_id(priv, values[0], &error_local);
```

Comment:
since dev is no longer you need to fix that:
```
../src/fu-util.c:1323:15: error: unused variable ‘dev’ [-Werror=unused-variable]
```

Reply:
Ups.. done!